### ISSUE TITLE
Falsy function doesn't seem to work

### ISSUE THREAD

**Issue by lehphyro** [stoplightio/spectral#1357]
**Describe the bug**
The falsy function doesn't seem to be working and I couldn't find built-in rules using it.

**To Reproduce**

1. Given this OpenAPI document:
```
openapi: 3.0.3
info:
  title: Test
  version: 1.0.0
  description: Description
  license:
    name: My license
    url: https://www.test.com/legal
paths:
  /test:
    get:
      summary: Test get
      description: Description
    responses:
```

2. And this linting rule:
```
no-license:
  description: “No license allowed”
  given: “$.info”
  severity: error
  then:
    field: license
    function: falsy
```

3. See that spectral still allows a license attribute

**Expected behavior**
A validation message saying that license attribute is not allowed.

Forum link: https://community.stoplight.io/t/falsy-function-doesnt-seem-to-work/1514

**Comment by P0lip** [comment]
@lehphyro 
Hey!
Just gave it a try on my end and it worked fine.
I can see the quotes you have are invalid, and perhaps this causes the issue.
```yaml
rules:
  no-license:
    description: "No license allowed"
    given: "$.info"
    severity: error
    then:
      field: license
      function: falsy
```
Does the above work well for you?

**Comment by lehphyro** [comment]
Ah yes, it worked with your sample rule. Sorry.
